subprojects/model-core/src/main/java/org/gradle/api/internal/provider/TransformBackedProvider.java

```diff
@@ -66,15 +66,11 @@ protected Value<? extends OUT> calculateOwnValue(ValueConsumer consumer) {
     }
 
     @Nonnull
-    private Value<? extends OUT> mapValue(Value<? extends IN> value) {
+    protected Value<? extends OUT> mapValue(Value<? extends IN> value) {
```

Comment:
I think this method should remain private.

Reply:
Right, fixed.